internal/gitstream/owners.go

```diff
@@ -0,0 +1,33 @@
+package gitstream
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+
+	"github.com/google/go-github/v47/github"
+)
+
+type Owners struct {
+	Approvers []string `yaml:"approvers"`
+	Reviewers []string `yaml:"reviewers"`
+	Component string   `yaml:"component"`
+}
+
+func (o *Owners) getAssignee(ctx context.Context, gc *github.Client, user *github.User) (*github.User, error) {
```

Comment:
I think this function could return just a `string` for the user login. That should remove the need for another call to `gc.Users.Get`.

Reply:
Same answer as for the previous question.